java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -455,14 +455,16 @@ private static Object[] geospatialCommands(BaseTransaction<?> transaction) {
                                 new GeospatialData(13.361389, 38.115556),
                                 "Catania",
                                 new GeospatialData(15.087269, 37.502669)))
-                .geopos(geoKey1, new String[] {"Palermo", "Catania"});
+                .geopos(geoKey1, new String[] {"Palermo", "Catania"})
+                .geohash(geoKey1, new String[] {"Palermo", "Catania", "NonExisting"});
 
         return new Object[] {
             2L, // geoadd(geoKey1, Map.of("Palermo", ..., "Catania", ...))
             new Double[][] {
                 {13.36138933897018433, 38.11555639549629859},
                 {15.08726745843887329, 37.50266842333162032},
             }, // geopos(new String[]{"Palermo", "Catania"})
+            new String[] {"sqc8b49rny0", "sqdtr74hyu0", null}
```

Comment:
Add a comment

Reply:
Applied the changes to the docs/comments in https://github.com/Bit-Quill/glide-for-redis/pull/275/commits/52d70471ed7790557d6ca681ecb9b6247dea9b50